lesson03.find-a-pair/js/memory_gui.js

```diff
@@ -0,0 +1,24 @@
+//////////////////////////////////////////////////////////////////////
+//                              GUI                                 //
+// drawBoard() -> re-draws the board and creates all html elements  //
+// showHideCard-> deals with showing front or back of cards         //
+//////////////////////////////////////////////////////////////////////
+function drawBoard() {
+    let htmlBuild = "";
+    for (i = 0; i < memoryGlobal.deck.length; i++) {
+        htmlBuild +=
+            `<div class="card" onclick="cardSelected(${i})">
+                        <img src="img/0.jpg">
+                        <img src="img/${memoryGlobal.deck[i].face}.jpg" class="hidden">
+                        </div>
+                    </div>`;
+    }
+    document.querySelector("main").innerHTML = htmlBuild;
+}
+
+function showHideCard(card, show = true) {
+    const selectedCard = document.querySelector('main').childNodes[card * 2];
```

Comment:
A better way would be to give each card a `class` or `data-attribute` and use query selector on it